src/api/services/userService.ts

```diff
@@ -105,21 +111,32 @@ export class UserService {
       {
         action: AuditAction.Update,
         email: user!.email, // So we know which user is being updated, in case their name changes
-        firstName: data.firstName,
-        lastName: data.lastName,
-        jobFunction: data.jobFunction,
+        firstName: userData.firstName,
+        lastName: userData.lastName,
+        jobFunction: userData.jobFunction,
+        userRoleIds: userRoleData.userRoleId,
       },
       participant!.id
     );
 
     await performAsyncOperationWithAuditTrail(auditTrailInsertObject, traceId, async () => {
       const kcAdminClient = await getKcAdminClient();
+      await UserToParticipantRole.query()
```

Comment:
couldn't find a better way to do this, given we can't update the table because the PK is all three columns.